repository/src/main/java/tech/ydb/yoj/repository/db/Table.java

```diff
@@ -270,54 +266,6 @@ default <V extends Table.View> ViewListResult<T, V> list(Class<V> viewType, List
         return ViewListResult.forPage(request, viewType, nextPage);
```

Comment:
We should also move `Table.toQueryBuilder(ListRequest)` to the query helper class